src/ui/components/TestSuite/views/TestRecording/TestRecordingEvents/UserActionEventRow.tsx

```diff
@@ -66,6 +71,14 @@ export default memo(function UserActionEventRow({
     replayClient
   );
 
+  const { value: domNode, status: domNodeStatus } = useImperativeCacheValue(
```

Comment:
You aren't using these values, so you should probably use `readAsync(...)` instead of `useImperativeCacheValue(...)` to avoid unnecessary rerenders.